react/ui-components/src/atoms/stories/Chip.stories.js

```
@@ -67,22 +78,22 @@ export const WithClose = Template.bind({});
+  hideClose: false,
 };
 
 export const WithIcon = Template.bind({});
 WithIcon.args = {
   ...commonArgs,
   text: "Chip",
-  iconReq:"Edit"
+  iconReq: "Edit",
 };
 
 export const Error = Template.bind({});
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)